godotopenxrkhr/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.godotengine.openxrloaders.khr"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <uses-permission android:name="org.khronos.openxr.permission.OPENXR_SYSTEM" />
+
+    <application>
+        <meta-data
+            android:name="org.godotengine.plugin.v1.GodotOpenXRKHR"
+            android:value="org.godotengine.openxrloaders.khr.GodotOpenXRKHR" />
+
+        <activity android:name="com.godot.game.GodotApp" android:exported="true" tools:ignore="MissingClass">
+            <intent-filter>
+                <!-- Mark this activity as using OpenXR. -->
+                <!-- action tag is here to make sure this intent filter is loaded by AOSP. -->
+                <action android:name="org.khronos.openxr.action.MAIN" />
+                <category android:name="org.khronos.openxr.intent.category.IMMERSIVE_HMD" />
```

Comment:
@rbessems Is this a valid category? If it's, can you add a comment that links to its description.

Also it seems the purpose of this intent-filter is to merge this category with Godot's `GodotApp` activity. The `GodotApp` activity is specific to the Godot Editor, so these changes should be made directly to the core Godot project's AndroidManifest.
 
The `<activity` should not be include in this loader library since it can be integrated by projects that don't include Godot's default `GodotApp` activity. Instead the documentation should be updated to inform users of this library to add the necessary category to their main activity.

Reply:
Yes valid: 

https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#android-runtime-category

Happy to put up another MR removing this. What is the best spot to add the documentation?

Reply:
It would be awesome if Godot adds this automatically for the default template,... wonder what the best way is to do that.

Reply:
Added a comment above describing the options you can use in core Godot to include this automatically. Let me know if you have questions.

Reply:
> What is the best spot to add the documentation?

It can be added both in the `README`, and in `GodotOpenXRKHR.java` as class javadocs (the javadocs is exported when this project is used as a library, so integrators can reference it).

@BastiaanOlij Any other place where the documentation should be included?

Reply:
I think the readme is the best location seeing this is of value to anyone who wishes to either add more to this repo or as we slowly improve the build system. 

We're probably too late to do major changes for 4.0 especially around the Android export in core, but as things evolve we'll want to refer back to decisions we've made here.